src/current/v23.1/import-performance-best-practices.md

```diff
@@ -7,61 +7,24 @@ docs_area: migrate
 
 This page provides best practices for optimizing [import](import-into.html) performance in CockroachDB.
 
-Import speed primarily depends on the amount of data that you want to import. However, there are two main factors that have can have a large impact on the amount of time it will take to run an import:
+Import speed primarily depends on the amount of data that you want to import. However, there are three actions you can take before importing that have a significant impact on the amount of time it will take to run an import:
 
-- [Splitting data](#split-your-data-into-multiple-files)
-- [Import format](#choose-a-performant-import-format)
+- [Choose a performant import format](#choose-a-performant-import-format)
+- [Split your data into multiple files](#split-your-data-into-multiple-files)
+- [Sort your data](#sort-your-data)
 
 {{site.data.alerts.callout_info}}
-If the import size is small, then you do not need to do anything to optimize performance. In this case, the import should run quickly, regardless of the settings.
+If the import size is small, then you do not need to do anything to optimize performance. For small datasets, the import should run quickly, regardless of the settings.
```

Comment:
Do we need to quantify small?

Reply:
I think so, but I don't know what the threshold is. My guess is somewhere in the tens of GBs of data.

Reply:
@jonstjohn do you have an opinion?

Reply:
I would say 100 GiB might be a good threshold. It depends on the number of nodes and CPU, but 100 GiB should go reasonably fast on any cluster, even a small one (e.g., 3 x 4 vCPU).